rascal2/ui/presenter.py

```diff
@@ -26,6 +27,7 @@ def createProject(self, name: str, save_path: str):
         """
 
         self.model.createProject(name, save_path)
+        self.view.setWindowTitle(self.title + " - " + name)
```

Comment:
Please just call `self.view.setWindowTitle(self.view.windowTitle() + " - " + name)` no need to store title in presenter

Reply:
This is the first thing I tried. This leads to a chain when new project is created. And I can't import MAIN_WINDOW_TITLE because it complains about circular import.

<img width="1023" alt="image" src="https://github.com/user-attachments/assets/1365fa43-0d41-43fe-8010-7d24b65c1c53">

Reply:
No problem, we can refactor later